### ISSUE TITLE
Add this role to Ansible Galaxy

### ISSUE THREAD

**Issue by nicorikken** [CISOfy/lynis-ansible#2]
Ansible Galaxy is great and easy for role discovery and role imports in general. Currently only the https://github.com/tommarshall/ansible-role-lynis role is available there.

**Comment by mboelen** [comment]
Galaxy asks too many regarding permissions when creating an account. For this reason we have not created such an account yet.

**Repositories**
Public only
This application will be able to read and write all public repository data. This includes the following:

Code
Issues
Pull requests
Wikis
Settings
Webhooks and services
Deploy keys

**Comment by nicorikken** [comment]
Fair enough 👍  Maybe a separate GitHub account can be created to prevent access into the other Lynis repo's? I added an upstream Galaxy ticket, hoping to change things for the better.
